app/assets/javascripts/helpers/requester.es6.jsx

```diff
@@ -53,6 +53,8 @@
           }
         }
       };
+      console.log(resolve);
```

Comment:
My listener doesn't get triggered after creating a response object. Printing out `resolve` in the console I get `return _this2.storeResponsibility(response);`. When I do the same procedure to create a comment I get `return _this.storeComment(response);`. Can the `2` have something to do with this?